### ISSUE TITLE
eodal mapper import breaks with pydantic version 2.0.2

### ISSUE THREAD

**Issue by gperich** [EOA-team/eodal#73]
The Mapper class of Eodal doesn't work with the newest pydantic version 2.0.2. 

**To Reproduce**
1. Install `eodal` using `pip install eodal`
2. Activate the python environment
3. In a python script; import the Mapper class: `from eodal.mapper.mapper import Mapper, MapperConfigs`

**Error Message**
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gperich/Projects/CropClassesTerensis/.venv/lib64/python3.11/site-packages/eodal/mapper/mapper.py", line 43, in <module>
    from eodal.config import get_settings
  File "/home/gperich/Projects/CropClassesTerensis/.venv/lib64/python3.11/site-packages/eodal/config/__init__.py", line 1, in <module>
    from .settings import get_settings
  File "/home/gperich/Projects/CropClassesTerensis/.venv/lib64/python3.11/site-packages/eodal/config/settings.py", line 35, in <module>
    from pydantic import BaseSettings
  File "/home/gperich/Projects/CropClassesTerensis/.venv/lib64/python3.11/site-packages/pydantic/__init__.py", line 206, in __getattr__
    return _getattr_migration(attr_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gperich/Projects/CropClassesTerensis/.venv/lib64/python3.11/site-packages/pydantic/_migration.py", line 279, in wrapper
    raise PydanticImportError(
pydantic.errors.PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.0.2/migration/#basesettings-has-moved-to-pydantic-settings for more details.
```

**Fix**
downgrade `pydantic` package to version 1.10.9
```
pip uninstall pydantic
pip install pydantic==1.10.9
```

**Comment by lukasValentin** [comment]
Apparently, we have to use `pydantic-settings` in the future (see [here](https://docs.pydantic.dev/2.0/migration/#basesettings-has-moved-to-pydantic-settings)) and update the import.

Since `pydantic` is only used in the `eodal.config.settings.py` module migrating from pydantic 1.x to pydantic 2.x should be not a big deal and will become part of the forthcoming EOdal release.

**Comment by lukasValentin** [comment]
I leave this issue open until we have made the new EOdal release (0.2.2) public. While the issue has been resolved in the master branch by #74, the problem still persists for people installing EOdal from pypi.

For those installing from pypi and experiencing the import-error described above, please use the fix provided by @gperich .

**Comment by lukasValentin** [comment]
##Important
In case you are using an out-dated version of `planetary-computer` make sure to also upgrade this package, as `planetary-computer` also uses `pydantic` and had the same issue as `EOdal`
